src/script.js

```diff
@@ -200,6 +200,10 @@ export function getPartnerAttributionID() : ?string {
     return getSDKAttribute(SDK_SETTINGS.PARTNER_ATTRIBUTION_ID);
 }
 
+export function getMerchantRequestedPopupDisabled() : boolean {
```

Comment:
I'm thinking we make `Popups` plural to align with the `POPUPS_DISABLED` constant

```suggestion
export function getMerchantRequestedPopupsDisabled() : boolean {
```